tests/test_config_utils.py

```diff
@@ -262,8 +253,6 @@ def test_autoconfigure_read_both_from_file_and_env(monkeypatch, request, ssl_con
     database = {}
     if DATABASE_BACKEND == 'mongodb':
```

Comment:
None of these backends exist now. The default one is now called `localmongodb`.

Reply:
This will change a lot of things. Will address in a separate PR.